embark/static/content/css/globalStyle.css

```diff
@@ -0,0 +1,133 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
+html {
+    --scrollbarBG: #d7e4ee;
+    --thumbBG: #b8cedb;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: 700;
+}
+
+body {
+    overflow-x: hidden;
+}
+
+.container {
+    position: fixed;
+    width: 100%;
+}
+
+.col {
+    max-width: 100%;
+    padding: 0;
+    margin: 0;
+}
+
+.container-fluid,
+.row {
+    padding: 0;
+    margin: 0;
+}
+
+/*Progress bars*/
+.progress {
+    height: 20px;
+    margin-top: 10px;
+    background-color: #b8cedb;
+    border-radius: 30px;
+}
+
+.progress-bar {
+    background-color: #009999;
+}
+
+/* text styles */
+.mainText {
+    font-size: large;
+    margin-bottom: 10px;
+}
+
+.subText {
+    font-size: medium;
+}
+
+.text {
+    font-size: medium;
+}
+
+/* button styles */
+.btn {
+    width: fit-content;
+    font-weight: normal;
+    border-radius: 30px;
+    font: 'arial';
+    line-height: 1;
+    font-size: medium;
+    color: #FFFFFF;
+    margin: 5px;
+    padding: 10px;
+}
+
+.btn:hover {
+    cursor: pointer;
+}
+
+/* submit button */
+button[type="submit"] {
+    background: #493791;
+    color: #FFFFFF;
+}
```

Comment:
does this overshadow button functionality for forms ?
e.g. if i got a form and want a submit button styled different to this, is this possible ?

Reply:
Is it the button used in django forms? or the button used similar to the one in uploader form?

Reply:
In the html we use something like this in the html to send the form:
```
<button type="submit" class="btn">Analyse</button>
```

Reply:
This is on a global level, so any button of the type "submit" these styles will be applied. To maintain uniformity I created this so that all submit or upload actions would have the same style. But if u want to override it you can in its respective style file using the class defined on the button. i.e in the eg u gave "btn" is the class. Make sure to have a unique class name and apply styles to it.

Reply:
Ok thank you for clarification.